src/store/rocket.js

```diff
@@ -0,0 +1,74 @@
+import _ from 'lodash'
+import { WebClient } from '@slack/web-api'
+
+const state = () => {
+    return {employeeHeadshots: null, 
+      employeesMissingFromSite: null
+    }
+  };
+  
+  const mutations = {
+    setEmployeeHeadshots(state, employeeHeadshots) {
+        state.employeeHeadshots = employeeHeadshots;
+      },
+    setEmployeesMissingFromSite(state, payload) {
+        const uniqueEmployees = _.differenceBy(payload, state.employeeHeadshots, 'name')
+        state.employeesMissingFromSite = uniqueEmployees;
+      }
+  };
+  
+  const actions = {
+    async getContentfulEmployees ({ commit }) {
+        try{
+            const employees = await this.$http.$get(this.$config.contentfulFullUrl)
+            const itemsFields = ['fields.name', 'fields.position', 'fields.profilePic.sys.id','sys.id']
+            const itemsFilteredEmployees = _.map(employees.items, e => _.pick(e, itemsFields))
+            const includesFields = ['fields.file.fileName', 'fields.file.url', 'sys.id']
+            const includesFilteredEmployees = _.map(employees.includes.Asset, e => _.pick(e, includesFields))
+            const mergedFilteredEmployees = _.values(_.merge(_.keyBy(includesFilteredEmployees,'sys.id'), _.keyBy(itemsFilteredEmployees,'fields.profilePic.sys.id' ) ))
```

Comment:
Ha! I just realized this would be a great exercise for a `_.reduce` function. See below if you haven't seen that comment yet :)